TaskHouseUnitTests/WorkersControllerUnitTests.cs

```diff
@@ -0,0 +1,217 @@
+using System;
+using Xunit;
+using TaskHouseApi.Controllers;
+using TaskHouseApi.Repositories;
+using Microsoft.AspNetCore.Mvc;
+using TaskHouseApi.Model;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
+namespace TaskHouseUnitTests
+{
+    public class WorkersControllerUnitTests
+    {
+        WorkersController controller;
+        IWorkerRepository repo;
+
+        public WorkersControllerUnitTests()
+        {
+            repo = new FakeWorkerRepository();
+            controller = new WorkersController(repo);
+        }
+
+        [Fact]
+        public async void WorkerController_Get_ReturnsObjectReponseWithCorrectEmpolyer_WhenGivenValidId()
+        {
+            Worker TestWorker = new Worker()
```

Comment:
New Worker objet isn't used, can simply be a variable referencing an existing ID, e.g. 1

Reply:
#cb85e64